### PR TITLE
add markdown to activity widget

### PR DESCRIPTION
**Pull request by kittybest**
add markdown styling to `ActivityWidget`

### CODE REVIEW

File: src/components/userPage/userPage.tsx [hunk 1]
```diff
@@ -411,7 +411,7 @@ const UserPage = () => {
                         <div>
                             {records.map((h, i) => (
                                 <ActivityWidget
-                                    key={h.data_id}
```
[the remark this answers is not in the document]
Comment (vimwitch) [reply src/components/userPage/userPage.tsx]: Why not use `transactionHash`? Timestamps can sometimes collide (though it's unlikely)

[the remark this answers is not in the document]
Comment (vimwitch) [reply src/components/userPage/userPage.tsx]: Ah we're not reading it in the `utils` `convertDataToRecord`. In a different pull request we should change this so the backend returns the data as we need it (so we don't have to format it on the frontend).

[the remark this answers is not in the document]
Comment (kittybest) [reply src/components/userPage/userPage.tsx]: ok, so just `h.time` now? and I'll make another PR for that.

[the remark this answers is not in the document]
Comment (vimwitch) [reply src/components/userPage/userPage.tsx]: Sure i think it's fine

File: src/pages/userPage/activityWidget.tsx [hunk 2]
```diff
@@ -76,29 +76,21 @@ const ActivityWidget = ({ record, isSpent }: Props) => {
...
-            }
-        } else {
+        if (record.content === undefined || record.content.length === 0)
             return { title: '', content: '' }
+
+        let i = record.content.indexOf(titlePrefix)
+        let j = record.content.indexOf(titlePostfix)
+        if (i === -1 || j === -1)
+            return { title: '', content: markdown.render(record.content) }
+        else {
```
Comment (vimwitch) [inline src/pages/userPage/activityWidget.tsx:86]: Don't need the else statement here 😅

**Pull request merged**